client/js/actions/view-action-creator.js

```diff
@@ -17,11 +17,11 @@ var ViewActionCreator = {
 
     ApiUtils.getDayData(date, ServerActionCreator.recieveDayData);
   },
-  getMonthEvents: function (date) {
+  getMonthData: function (date) {
     AppDispatcher.dispatch({
-      actionType: AppConstants.ViewActionTypes.GET_MONTH_EVENTS
+      actionType: AppConstants.ViewActionTypes.GET_MONTH_Data
```

Comment:
Good catch, looks like auto correct finished that one off for me and it passed my testing because chrome cached the JS file! Will fix and update